specs/core/0_beacon-chain.md

```diff
@@ -601,6 +601,35 @@ def is_slashable_validator(validator: Validator, epoch: Epoch) -> bool:
     return (not validator.slashed) and (validator.activation_epoch <= epoch < validator.withdrawable_epoch)
 ```
 
+#### `is_eligible_for_activation_queue`
```

Comment:
Need to update ToC, or clean up together in #1512.

Reply:
done